public/index.html

```diff
@@ -185,6 +185,52 @@
 
     <hr/>
 
+		<uxtabarea>
+			<uxtablinkcontainer>
+				<uxtablink controls="firstTab" active="true">Tab One</uxtablink>
+				<uxtablink controls="secondTab">Tab Two</uxtablink>
+				<uxtablink controls="thirdTab">Tab Three</uxtablink>
```

Comment:
Setting it explicitly provides flexibility in how you structure it. Bringing it internal will require a strict order. I'm not so sure we want to force that on people? Obviously it _should_ be rare that they are out of order.